docs/src/components/StorybookAdminComponentDocsIframe.tsx

```diff
@@ -0,0 +1,66 @@
+import BrowserOnly from "@docusaurus/BrowserOnly";
+import { useEffect, useRef, useState } from "react";
+
+type Props = {
+    title: string;
+};
+
+const fallbackIframeHeight = 800;
+
+export const StorybookAdminComponentDocsIframe = ({ title }: Props) => {
+    return <BrowserOnly>{() => <IFrame title={title} />}</BrowserOnly>;
+};
+
+const IFrame = ({ title }: Props) => {
+    const pathName = title.toLowerCase();
+    const storybookDomain = getStorybookDomain();
+    const storybookDocsUrl = `${storybookDomain}/iframe.html?viewMode=docs&id=component-docs-${pathName}--docs&isEmbeddedInDocs=true`;
+    const { iframeContentHeight, iframeRef } = useIframeContentHeight();
+
+    return <iframe src={storybookDocsUrl} width="100%" height={iframeContentHeight} ref={iframeRef} />;
+};
+
+const useIframeContentHeight = () => {
```

Comment:
I guess it isn't possible to have a non-JS-solution, right?

Should we use something like https://github.com/davidjbradshaw/iframe-resizer, but with a more permissive license?

Reply:
> I guess it isn't possible to have a non-JS-solution, right?

No 🙂 

> Should we use something like https://github.com/davidjbradshaw/iframe-resizer, but with a more permissive license?

I tried using a free alternative but didn't get it to work and didn't want to waste more time trying. 
Basically all those libraries just use `postMessage()` in the child-frame and then listen to the message in the parent. That seems quite straight forward to me and I don't really see the benefit in using a library if it doesn't just work out of the box.